### ISSUE TITLE
expect(tree).to.matchSnapshot(true); don't update the snapshot and adds "true" string to the end of the snapshot name

### ISSUE THREAD

**Issue by bertho-zero** [suchipi/chai-jest-snapshot#32]
I use *Jest* with *Enzyme* and when I use the following line like in the doc for updating my snapshot, that creates 1 more snapshot with the same name plus the `true` string at the end:
```js
expect(tree).to.matchSnapshot(true);
```
---
This test name
```
conversation destinated to a user 1
```
became:
```
conversation destinated to a user: true 1
```

**Comment by suchipi** [comment]
Ah, this should be fixed in the documentation. When using jest, the arguments to `matchSnapshot` are ignored- you should use `jest -u` to update your snapshots instead.

**Comment by bertho-zero** [comment]
Okay, that's a shame, because it would allow to quickly update one or more snapshots without going through the parameters from Jest (--testNamePattern, --testPathPattern).

**Comment by suchipi** [comment]
Yeah. I'll keep this open and maybe later I'll see if it's possible to tell jest to update a single snapshot, and if it is, I'll make this behave like Mocha/etc.
